nevergrad/benchmark/plotting.py

```diff
@@ -435,7 +449,11 @@ def __init__(
         self._ax.grid(True, which="both")
         self._overlays: tp.List[tp.Any] = []
         legend_infos: tp.List[LegendInfo] = []
-        for optim_name in sorted_optimizers:
+        for optim_name in (
+            sorted_optimizers[:1] + sorted_optimizers[-12:]
```

Comment:
the first one and the last 12? I dont get this

Reply:
This includes the 12 best, and the worst method.

This is for having an eye on bias in noisy optimization. 

Consider a case in which the expected loss value is always >= 0 (equal to 0 at the optimum).
In the noisy case, you will still get a sudden decrease of the 12 best methods at the last x-axis value: because we select methods with the best losses at the last x-axis value.

With also the worst, you get something more interpretable: you see that this is just noise.